Storage/SQL/SQLiteHistoryFavicons.swift

```diff
@@ -15,20 +14,9 @@ public let DefaultFaviconBackgroundColors = ["2e761a", "399320", "40a624", "57bd
 
 private let log = Logger.syncLogger
 
-// Set up Alamofire for downloading web content for parsing.
+// Set up for downloading web content for parsing.
 // NOTE: We use the desktop UA to try and get hi-res icons.
-private var alamofire: SessionManager = {
-    var sessionManager: SessionManager!
-    DispatchQueue.main.sync {
-        let configuration = URLSessionConfiguration.default
-        var defaultHeaders = SessionManager.default.session.configuration.httpAdditionalHeaders ?? [:]
-        defaultHeaders["User-Agent"] = UserAgent.desktopUserAgent()
-        configuration.httpAdditionalHeaders = defaultHeaders
-        configuration.timeoutIntervalForRequest = 5
-        sessionManager = SessionManager(configuration: configuration)
-    }
-    return sessionManager
-}()
+private var urlSession: URLSession = makeUrlSession(userAgent: UserAgent.desktopUserAgent(), isEphemeral: false, timeout: 5)
```

Comment:
`UserAgent.desktopUserAgent()` needs to be called on the main thread.

Reply:
Yes, all of the non-db code in the app is main-thread only. Can you clarify why you mentioned this?

Reply:
Testing this on my device causes an assertionFailure when loading any panels with favicon. `Assertion failed: This method must be called on the main thread.: file firefox-ios/Shared/UserAgent.swift, line 62`

Reply:
thanks! i'll take a look